src/app/store.ts

```diff
@@ -1,6 +1,14 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
Ensure that `combineSlices` is the correct function to use here. Typically, `combineReducers` is used in Redux Toolkit. Verify that `combineSlices` is intended and correctly imported.